packages/ui/src/1_atoms/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactNode, useMemo } from 'react';
+
+import classNames from 'classnames';
+
+import styles from './Paragraph.module.css';
+import {
+  ParagraphSize,
+  ParagraphStyle,
+  ParagraphType,
+} from './Paragraph.types';
+
+export type ParagraphProps = {
+  children: ReactNode;
+  type?: ParagraphType;
+  size?: ParagraphSize;
+  style?: ParagraphStyle;
+  className?: string;
+};
+
+export const Paragraph: React.FC<ParagraphProps> = ({
+  children,
+  type = ParagraphType.regular,
+  size = ParagraphSize.base,
+  style = ParagraphStyle.normal,
+  className,
+}) => {
+  const classNameComplete = useMemo(
+    () =>
+      classNames(
+        styles.paragraph,
+        type && styles[type],
+        size && styles[size],
+        style && styles[style],
+        className,
+      ),
+    [size, style, type, className],
+  );
```

Comment:
```suggestion
  const classNames = useMemo(
    () =>
      classNames(
        styles.paragraph,
        styles[type],
        styles[size],
        styles[style],
        className,
      ),
    [size, style, type, className],
  );
```

Reply:
the classNames const will conflict with classNames from the classnames library, so will leave it as it is.